db/crud.go

```diff
@@ -1901,6 +1986,27 @@ func (db *DatabaseCollectionWithUser) resolveDocMerge(ctx context.Context, local
 	return mergedRevID, docHistory, nil
 }
 
+// resolveLocalWinsHLV will update remote doc's body and attachments to match the local doc, and return a new HLV for local wins
+func (db *DatabaseCollectionWithUser) resolveLocalWinsHLV(ctx context.Context, localDoc, remoteDoc *Document) (*HybridLogicalVector, error) {
```

Comment:
There seem to be a few differences between `resolveLocalWinsHLV` and `resolveDocLocalWins`, some of which seem right, and some of which seem not right.

1. There's not a new revtree id that is created in this function. You would need a local revtree id to support 3.x CBL replication even after 4.x ISGR replication.
2. There's handling of branching tombstones. I'm not sure there will ever be branching since the local revtree is already overwritten by remote revtree by the time we are here?  but I'm actually not sure.
3. tombstoneActiveRevision means that the old revision is added to the doc.History for revtree ids.